### PR TITLE
Chore: Fix lint issues and add indent with tabs

### PR DESCRIPTION
**Pull request by gyandeeps**
* Fix 2 existing lint issues
* Add `indent` rule with `tabs` options (as per `.editorconfig`)
* Add fix script to quickly fix issue
    * I used it in this PR to fix indent tab issues

### CODE REVIEW

File: render/render.js [hunk 1]
```diff
@@ -24,10 +24,10 @@ module.exports = function($window) {
 			vnode.state = {}
 			if (vnode.attrs != null) initLifecycle(vnode.attrs, vnode, hooks)
 			switch (tag) {
-				case "#": return createText(parent, vnode, nextSibling)
-				case "<": return createHTML(parent, vnode, nextSibling)
-				case "[": return createFragment(parent, vnode, hooks, ns, nextSibling)
-				default: return createElement(parent, vnode, hooks, ns, nextSibling)
+			case "#": return createText(parent, vnode, nextSibling)
```
Comment (tivac) [inline render/render.js]: These should be indented another level.

File: render/render.js [hunk 2]
```diff
@@ -67,8 +67,8 @@ module.exports = function($window) {
 	function createElement(parent, vnode, hooks, ns, nextSibling) {
 		var tag = vnode.tag
 		switch (vnode.tag) {
-			case "svg": ns = "http://www.w3.org/2000/svg"; break
-			case "math": ns = "http://www.w3.org/1998/Math/MathML"; break
+		case "svg": ns = "http://www.w3.org/2000/svg"; break
```
Comment (tivac) [inline render/render.js]: These should be indented another level.

File: render/render.js [hunk 3]
```diff
@@ -246,10 +246,10 @@ module.exports = function($window) {
 					else updateLifecycle(vnode.attrs, vnode, hooks)
 				}
 				switch (oldTag) {
-					case "#": updateText(old, vnode); break
-					case "<": updateHTML(parent, old, vnode, nextSibling); break
-					case "[": updateFragment(parent, old, vnode, recycling, hooks, nextSibling, ns); break
-					default: updateElement(old, vnode, recycling, hooks, ns)
+				case "#": updateText(old, vnode); break
```
Comment (tivac) [inline render/render.js]: These should be indented another level.

File: render/render.js [hunk 4]
```diff
@@ -290,8 +290,8 @@ module.exports = function($window) {
 	function updateElement(old, vnode, recycling, hooks, ns) {
 		var element = vnode.dom = old.dom
 		switch (vnode.tag) {
-			case "svg": ns = "http://www.w3.org/2000/svg"; break
-			case "math": ns = "http://www.w3.org/1998/Math/MathML"; break
+		case "svg": ns = "http://www.w3.org/2000/svg"; break
```
Comment (tivac) [inline render/render.js]: These should be indented another level.

File: router/router.js [hunk 5]
```diff
@@ -44,9 +44,9 @@ module.exports = function($window) {
 	router.getPath = function() {
 		var type = router.prefix.charAt(0)
 		switch (type) {
-			case "#": return normalize("hash").slice(router.prefix.length)
-			case "?": return normalize("search").slice(router.prefix.length) + normalize("hash")
-			default: return normalize("pathname").slice(router.prefix.length) + normalize("search") + normalize("hash")
+		case "#": return normalize("hash").slice(router.prefix.length)
```
Comment (tivac) [inline router/router.js]: These should be indented another level.

File: stream/tests/test-stream.js [hunk 6]
```diff
@@ -152,12 +152,10 @@ o.spec("stream", function() {
 			o(b()()).equals(undefined)
 		})
 		o("combine can halt", function() {
-			var count = 0
 			var a = Stream(1)
 			var b = Stream.combine(function() {
 				return Stream.HALT
 			}, [a])["fantasy-land/map"](function() {
-				count++
```
Comment (pygy) [inline stream/tests/test-stream.js]: This is a missed `count === 0` assertion, actually.

File: api/tests/test-router.js [hunk 7]
```diff
@@ -507,7 +507,6 @@ o.spec("route", function() {
 				})
 
 				o("accepts RouteResolver without `onmatch` method as payload", function() {
-					var renderCount = 0
```
Comment (pygy) [inline api/tests/test-router.js]: Likewise, this is a missed assertion for `renderCount === 1`, it should not be removed, but an assertion added.

File: render/render.js (hunk 1 above)
Comment (gyandeeps) [inline render/render.js]: Woops.. sorry forgot to add `swicth: 1` to the rule.

File: stream/tests/test-stream.js (hunk 6 above)
Comment (gyandeeps) [inline stream/tests/test-stream.js]: I will add the assertions. But i removed these lines and they were not removed automatically. Eslint fix safe when it comes to auto fixing.

**Comment by gyandeeps** [comment]
@tivac and @pygy Done 😄

**Comment by pygy** [comment]
@gyandeeps Thanks!

ESLint has had not-that safe fixes in the past, hence my concern

**Pull request merged**